docs/pages/application-access/troubleshooting-apps.mdx

```diff
@@ -153,3 +153,17 @@ This configuration is available under the `jwt_claims` property of the
 application's `rewrite` configuration. See
 [Web Application Access](./guides/connecting-apps.mdx#configuring-the-jwt-token)
 for details.
+
+## Request too large
+
+Teleport enforces a 10MB size limit for the body of each HTTP request.
```

Comment:
do we enforce the 10MB on regular HTTP app? i though we only rewrite body for AWS/Azure/GCP.

Reply:
hmm, I'll check. I thought we enforced this in a lot of places